docs/docs-beta/docs/dagster-plus/deployment/hybrid/agents/kubernetes.md

```diff
@@ -1,8 +1,81 @@
 ---
-title: "Kubernetes agents"
+title: "Running Dagster+ agents on Kubernetes"
 displayed_sidebar: "dagsterPlus"
 sidebar_position: 30
 sidebar_label: "Kubernetes"
 ---
 
-# Kubernetes agents
\ No newline at end of file
+# Running Dagster+ agents on Kubernetes
+
+This page provides instructions for running the [Dagster+ agent](dagster-plus/getting-started/whats-dagster-plus#Agents) on a [Kubernetes](https://kubernetes.io) cluster.
+
+## Installation
+
+
+### Prerequisites
+
+You'll need [a Kubernetes cluster. This can be a self-hosted Kubernetes cluster or a managed offering like [Amazon EKS](https://aws.amazon.com/eks/), [Azure AKS](https://azure.microsoft.com/en-us/products/kubernetes-service), or [Google GKE](https://cloud.google.com/kubernetes-engine).
```

Comment:
```suggestion
You'll need a Kubernetes cluster. This can be a self-hosted Kubernetes cluster or a managed offering like [Amazon EKS](https://aws.amazon.com/eks/), [Azure AKS](https://azure.microsoft.com/en-us/products/kubernetes-service), or [Google GKE](https://cloud.google.com/kubernetes-engine).
```